cmd/juju/controller/addmodel.go

```diff
@@ -55,20 +55,30 @@ type addModelCommand struct {
 
 const addModelHelpDoc = `
 Adding a model is typically done in order to run a specific workload.
-To add a model, you must at a minimum specify a model name. You may
-also supply model-specific configuration, a credential, and which
-cloud/region to deploy the model to. The cloud/region and credentials
-are the ones used to create any future resources within the model.
 
+To add a model, you must specify a model name. 
 Model names can be duplicated across controllers but must be unique for
 any given controller. Model names may only contain lowercase letters,
 digits and hyphens, and may not start with a hyphen.
 
-Credential names are specified either in the form "credential-name", or
-"credential-owner/credential-name". There is currently no way to acquire
-access to another user's credentials, so the only valid value for
-credential-owner is your own user name. This may change in a future
-release.
+To add a model, Juju requires a credential.
+If a controller has a credential for the user creating the model, 
```

Comment:
unspecified-credential only applies to the admin user

- if you have a default (or just one) credential defined at client (i.e. in credentials.yaml), then juju will use that
- if you have no default (and multiple) credentials defined at the client, then you must specify one using --credential
- as the admin user you can omit the credential, and the credential used to bootstrap will be used